glean_parser/pings.py

```diff
@@ -73,3 +73,9 @@ def serialize(self) -> Dict[str, util.JSONType]:
         d = self.__dict__.copy()
         del d["name"]
         return d
+
+    def identifier(self) -> str:
```

Comment:
pings.py? Shouldn't this be on Metric, not Ping?

Reply:
The same template is used to generate ping objects and metric objects, so we need `identifier` defined on both.  (Alternatively, could have put this logic in the templates, but that was much more complex...)